### PR TITLE
[PSM Interop] Add bootstrap generator test

### PR DESCRIPTION
**Pull request by yashykt**
Based on https://github.com/grpc/grpc-go/pull/6463

### CODE REVIEW

**Review by sergiitk — CHANGES_REQUESTED** [review]
The change this is based on does not work.  We'll debug it there.

File: tools/internal_ci/linux/grpc_xds_k8s_lb.sh [hunk 1]
```diff
@@ -180,6 +180,9 @@ main() {
     "remove_neg_test"
     "round_robin_test"
   )
+  if [[ "${TESTING_VERSION}" == "master" ]]; then
```
Comment (zasweq) [inline tools/internal_ci/linux/grpc_xds_k8s_lb.sh:183]: This works now we merged the fix. Switch this == to ~=.

**Review by zasweq — CHANGES_REQUESTED** [review]

File: tools/internal_ci/linux/grpc_xds_k8s_lb.sh (hunk 1 above)
Comment (yashykt) [reply tools/internal_ci/linux/grpc_xds_k8s_lb.sh]: Alright, done

**Comment by zasweq** [comment]
Thanks, now I just need to trigger a manual run of these to make sure it works.

**Comment by sergiitk** [comment]
- [ ] [grpc/java/master/branch/xds_k8s_lb](https://source.cloud.google.com/results/invocations/2d8ca224-0771-4592-8fee-eafe038fc8a0)

**Comment by zasweq** [comment]
It worked! This should be ok to merge now.

**Review by sergiitk — CHANGES_REQUESTED** [review]
LGTM, but we need to increase the timeout for this job. The successful test run took 2 hr, 42 min. Let's bump it up to 4 hours:
https://github.com/grpc/grpc/blob/4111738c71ec2e31f884df28aabb185a4b78c0f6/tools/internal_ci/linux/grpc_xds_k8s_lb.cfg#L19

**Comment by yashykt** [comment]
Done

**Review by sergiitk — APPROVED** [review]

**Pull request merged**